client/src/pages/PatientsPage.jsx

```diff
@@ -1,9 +1,17 @@
-import React, { useState } from 'react';
-import patients from '../../patients.json';
+import React, { useState, useEffect } from 'react';
+// import patients from '../../patients.json';
 import ObjectsListing from '../components/listing/ObjectsListing';
 import SearchBar from '../components/SearchBar';
 
 const PatientsPage = () => {
+  const [patients, setPatients] = useState([]);
+
+  useEffect(() => {
+    fetch('https://hms.test/api/v1/patients/getall/db')
```

Comment:
It will fetch all users from db, i suggest to make something like this:
`fetch(PATIENTS_BASE_URL + "/getall/db"`